### PR TITLE
[next-devel] overrides: fast-track frozen packages to avoid downgrades

### PR DESCRIPTION
**Pull request by saqibali-2k**
```
$ rpm-ostree db diff fedora-compose:fedora/x86_64/coreos/{testing,next}-devel

...

Downgraded:
  containerd 1.5.7-1.fc34 -> 1.5.5-1.fc35
  kernel 5.14.13-200.fc34 -> 5.14.11-300.fc35
  kernel-core 5.14.13-200.fc34 -> 5.14.11-300.fc35
  kernel-modules 5.14.13-200.fc34 -> 5.14.11-300.fc35
  libgusb 0.3.8-1.fc34 -> 0.3.7-2.fc35
  libxmlb 0.3.3-1.fc34 -> 0.3.2-2.fc35
  moby-engine 20.10.9-1.fc34 -> 20.10.8-1.fc35
  ostree 2021.5-2.fc34 -> 2021.4-2.fc35
  ostree-libs 2021.5-2.fc34 -> 2021.4-2.fc35
  rpm-ostree 2021.12-2.fc34 -> 2021.11-3.fc35
  rpm-ostree-libs 2021.12-2.fc34 -> 2021.11-3.fc35
  tzdata 2021c-1.fc34 -> 2021b-1.fc35
```

### CODE REVIEW

File: manifest-lock.overrides.yaml [hunk 1]
```diff
@@ -46,29 +52,71 @@ packages:
...
+      reason: https://github.com/coreos/fedora-coreos-config/pull/1305#issuecomment-951242424
+      type: fast-track   
   skopeo:
     evr: 1:1.5.0-2.fc35
     metadata:
       bodhi:  https://bodhi.fedoraproject.org/updates/FEDORA-2021-982b9a5061
       reason: https://github.com/coreos/fedora-coreos-streams/issues/383#issuecomment-946115687
       type: fast-track
+  tzdata:                                                                   
+    evr: 2021c-1.fc35
```
Comment (dustymabe) [inline manifest-lock.overrides.yaml]:
```suggestion
    evra: 2021c-1.fc35.noarch
```

**Review by dustymabe — CHANGES_REQUESTED** [review]
one suggested change otherwise LGTM

File: manifest-lock.overrides.yaml (hunk 1 above)
Comment (saqibali-2k) [reply manifest-lock.overrides.yaml]: Fixed!

**Comment by saqibali-2k** [comment]
Hmm not sure what this means:
```
[2021-10-26T00:08:58.295Z] Using RPM overrides from: /srv/fcos/overrides/rpm

[2021-10-26T00:08:58.295Z] Running: rpm-ostree compose tree --repo=/srv/fcos/tmp/repo --touch-if-changed /srv/fcos/tmp/treecompose.changed --cachedir=/srv/fcos/cache --unified-core /srv/fcos/tmp/override/coreos-assembler-override-manifest.yaml --download-only --ex-lockfile=/srv/fcos/src/config/manifest-lock.x86_64.json --ex-lockfile=/srv/fcos/src/config/manifest-lock.overrides.yaml --ex-lockfile-strict --ex-lockfile=/srv/fcos/tmp/override/local-overrides.json

[2021-10-26T00:08:58.295Z] info: Missing CAP_SYS_ADMIN; using virt

[2021-10-26T00:08:58.295Z] Formatting 'cache2.qcow2.tmp', fmt=qcow2 cluster_size=65536 extended_l2=off compression_type=zlib size=10737418240 lazy_refcounts=off refcount_bits=16

[2021-10-26T00:09:16.381Z] error: Parsing /srv/fcos/src/config/manifest-lock.overrides.yaml: serde-yaml: packages: data did not match any variant of untagged enum LockedPackage at line 12 column 19
```

**Comment by dustymabe** [comment]
> Hmm not sure what this means:

took me a while but I found it!

File: manifest-lock.overrides.yaml [hunk 2]
```diff
@@ -46,29 +52,71 @@ packages:
...
+    evr: 5.14.13-300.fc35
     metadata:
-      bodhi:  https://bodhi.fedoraproject.org/updates/FEDORA-2021-f377355e41
-      reason: https://github.com/coreos/fedora-coreos-streams/issues/383#issuecomment-946115687
+      bodhi: https://bodhi.fedoraproject.org/updates/FEDORA-2021-0f33f94e81
+      reason: https://github.com/coreos/fedora-coreos-config/pull/1305#issuecomment-951242424
       type: pin
+  libgusb:
+    evr: 0.3.8-1.fc35
+    metadta:
```
Comment (dustymabe) [inline manifest-lock.overrides.yaml]:
```suggestion
    metadata:
```

Comment (saqibali-2k) [reply manifest-lock.overrides.yaml]: ah shoot sorry about that, thanks for finding it!

**Review by dustymabe — APPROVED** [review]

**Pull request merged**